examples/webgl/src/main.rs

```diff
@@ -45,44 +27,41 @@ impl Component for App {
         }
     }
 
-    fn rendered(&mut self, ctx: &Context<Self>, first_render: bool) {
+    fn rendered(&mut self, _ctx: &Context<Self>, first_render: bool) {
+        // Only start the render loop if it's the first render
+        // There's no loop cancellation taking place, so if multiple renders happen,
+        // there will be multiple loops running. That doesn't *really* matter here because
```

Comment:
```suggestion
        // there would be multiple loops running. That doesn't *really* matter here because
```

An unfortunate typo, but one that confuses the meaning and contradicts the previous sentence.